apps/artifactory/sbox-intsvc/base/kustomization.yaml

```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 resources:
   - ../../base
   - ../../artifactory/sbox-intsvc-ingress.yaml
```

Comment:
You will need to remove this ingress patch as you've got the ingress settings included in your chart

Reply:
Ingress in this PR is a slightly different host name `  - artifactory-helm.sandbox.platform.hmcts.net` - think it could deploy

Reply:
think it depend how the ingress created by the chart is named/its metadata right? Much easier to avoid and just update in place as mentioned in comment above